src/sentry/metrics/datadog.py

```diff
@@ -50,9 +43,8 @@ def incr(self, key, instance=None, tags=None, amount=1, sample_rate=1):
             tags['instance'] = instance
         if tags:
             tags = ['{}:{}'.format(*i) for i in tags.items()]
-        # datadog does not implement sampling here
-        amount = _sampled_value(amount, sample_rate)
```

Comment:
Ultimately, the problem here is that we aren't sampling the # of calls to `backend.incr()` first. So we execute this function for every call, but yet we're adjusting the `amount` to account for the fact that it was called less frequently, resulting in the value to be inflated instead.